static/js/compose_validate.js

```diff
@@ -654,6 +655,81 @@ function validate_private_message() {
     return true;
 }
 
+export function check_send_permissions() {
+    const send_button = document.querySelector("#compose-send-button");
+    const send_button_container = document.querySelector(".compose_right_float_container");
```

Comment:
Any problem in directly selecting element as $("#compose-send-button"), since that what we do majorly. And then we can change other code below too accordingly.

Reply:
This has now been updated in the latest commit to follow the $(...) pattern